docs-site/content/guide/laravel-full-text-search.md

```diff
@@ -0,0 +1,567 @@
+                        ],
+                        [
+                            "name" => "hltb_single",
+                            "type" => "int32",
+                            "facet" => true,
+                            "optional" => true,
+                        ],
+                        [
+                            "name" => "positive",
+                            "facet" => true,
+                            "type" => "int32",
+                        ],
+                        [
+                            "name" => "negative",
+                            "facet" => true,
+                            "type" => "int32",
+                        ],
+                        [
+                            "name" => "app_id",
+                            "type" => "string",
+                        ],
+                        [
+                            "name" => "min_owners",
+                            "type" => "int32",
+                        ],
+                        [
+                            "name" => "max_owners",
+                            "type" => "int32",
+                        ],
+                        [
+                            "name" => "created_at",
+                            "type" => "int64",
+                        ],
+                        [
+                            "name" => "release_date",
+                            "type" => "int64",
+                        ],
+                    ],
+                    "default_sorting_field" => "release_date",
+                ],
+                "search-parameters" => [
+                    "query_by" => "name, app_id",
+                ],
+            ],
+        ],
+    ],
+```
+
+</template>
+
+</Tabs>
+
+After which, you can import the data into Typesense by running the following command:
+
+<Tabs :tabs="['Shell']">
+
+<template v-slot:Shell>
+
+```shell
+./vendor/bin/sail artisan scout:import "App\Models\Game"
+```
+  
+</template>
+
+</Tabs>
+
+To test that everything is working correctly, you can run a search query:
+
+<Tabs :tabs="['Shell', 'PHP']">
+
+<template v-slot:Shell>
+
+```shell
+curl -H "X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}" \
+"http://localhost:8108/collections/games/documents/search\
+?q=persona&query_by=name"
+```
+
+</template>
+
+<template v-slot:PHP>
+
+```php
+use App\Models\Game;
+
+Game::search('persona')->get()->toArray();
+```
+
+</template>
+
+</Tabs>
+
+
+You can then proceed as you prefer. You can create a controller to handle the search requests, or use the [Typesense InstantSearch Adapter](https://github.com/typesense/typesense-instantsearch-adapter) to use Instantsearch.js on your frontend. If you prefer using a Javascript framework, you can use [Inertia.js](https://inertiajs.com/) to create a Vue.js, Svelte or React.js frontend. This [demo](https://github.com/typesense/showcase-laravel-steam-games-search/tree/master) uses a React Typescript frontend with the Typesense InstantSearch Adapter.
```

Comment:
Let's move the demos to under a new heading called "Demos".

Reply:
Could you please elaborate on this a bit more? I should add a new Heading called demos and include just the part that references the Demo, or move the whole paragraph over there?

Reply:
Yeah, a new Heading called "Demo Apps" and include just the part that references the demo to it. This way it stands out in the sidenav bar

Something like this: https://typesense.org/docs/26.0/api/vector-search.html#ui-examples

Reply:
> Yeah, a new Heading called "Demo Apps" and include just the part that references the demo to it. This way it stands out in the sidenav bar
> 
> Something like this: https://typesense.org/docs/26.0/api/vector-search.html#ui-examples

Addressed in 2ec91527da8972b0fd26cc48488785b60d19e1da